src/components/Home.js

```diff
@@ -0,0 +1,45 @@
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import america from '../images/america.png';
+import Footer from './Footer';
+
+const Home = () => {
+  const fetchNa = useSelector((state) => state.northAmerica);
+  const fetchCa = useSelector((state) => state.centralAmerica);
+  const fetchSa = useSelector((state) => state.southAmerica);
+  const fetchTc = useSelector((state) => state.theCaribbean);
+  const na = fetchNa;
+  const ca = fetchCa;
+  const sa = fetchSa;
+  const tc = fetchTc;
+
+  return (
+    <div className="wrapper">
+      <div className="homeContainer">
+        <img id="america" src={america} alt="american-continent" />
+        <div className="mainTitle">
+          <h1>Cities</h1>
+          <p>Total: 56</p>
+        </div>
+      </div>
+      <p id="stats">Select capital city by region</p>
+      <div className="regionCont">
+        <Link to="/details_north_america">
+          <div role="button" tabIndex={0} className="na">{`North America ${na.length} capital cities`}</div>
+        </Link>
+        <Link to="/details_central_america">
+          <div role="button" tabIndex={0} className="ca">{`Central America ${ca.length} capital cities`}</div>
+        </Link>
+        <Link to="/details_south_america">
+          <div role="button" tabIndex={0} className="sa">{`South America ${sa.length} capital cities`}</div>
+        </Link>
+        <Link to="/details_the_caribbean">
+          <div role="button" tabIndex={0} className="tc">{`The Caribbean ${tc.length} capital cities`}</div>
+        </Link>
+      </div>
+      <Footer />
+    </div>
+  );
```

Comment:
- [ ] You are yet to implement the filtering functionality as listed in the guidelines. Kindly do that.

- [ ] Also ensure this change is reflected in the live demo.

  ![wuye-1](https://user-images.githubusercontent.com/36542986/176770569-e59ea59b-5b1b-4b2a-a171-367d5e11a726.jpg)

Reply:
Thanks for this comment TedLivist, in this particular case I think this does not apply to my project because I'm already doing that in the main redux store. In other words, I have a reducer for each region and assign this reducer to configureStore()

I have a reducer for each region: North America, south-America and so on. Then I retrieve the global store with useSelector() and "filter" items by region index.
![image](https://user-images.githubusercontent.com/58822719/176774812-2082ba22-16be-439d-b77b-fd63a88df60f.png)

I'm fetching the data for the store using useSelector() and "filtering" by region in each reducer:
![image](https://user-images.githubusercontent.com/58822719/176775109-172dd079-015d-4e69-9db8-60ee2578e5f2.png)